actors/miner/src/lib.rs

```diff
@@ -3300,6 +3300,12 @@ impl Actor {
                 if info.beneficiary != info.owner {
                     // remaining_quota always zero and positive
                     let remaining_quota = info.beneficiary_term.available(rt.curr_epoch());
+                    if remaining_quota.is_zero() {
+                        return Err(actor_error!(
+                            illegal_state,
+                            "beneficiary expired or quota depleted. change or update beneficiary to withdraw funds"
```

Comment:
periods in error messages aren't canonical

Reply:
print the beneficiary's expire epoch and used quota/quota to help the user determine what is failing